logging/bin/deploy_fluentbit_azmonitor.sh

```diff
@@ -61,13 +61,21 @@ fi
 
 
 # Fluent Bit
+# TO DO: Discuss; needed? better? works?
+# Remove existing Helm chart, if it exists
+#if [ helm status -n $LOG_NS fbaz 1>/dev/null 2>&1 ]; then
+#   log_debug "Remove Existing Fluent Bit Release"
+#   helm delete -n $LOG_NS fbaz
+#fi
+
 # Create ConfigMap containing Fluent Bit configuration
 kubectl -n $LOG_NS apply -f $FB_CONFIGMAP
 
 # Create ConfigMap containing Viya-customized parsers (delete it first)
 kubectl -n $LOG_NS delete configmap fbaz-viya-parsers --ignore-not-found
 kubectl -n $LOG_NS create configmap fbaz-viya-parsers  --from-file=logging/fb/viya-parsers.conf
 
+# TO DO: Replace with Helm release removal?
 # Delete any existing Fluent Bit pods in the $LOG_NS namepace (otherwise Helm chart may assume an upgrade w/o reloading updated config
```

Comment:
Since the pods are managed as a daemonset, deleting the pods won't do much, right?

Reply:
The delete pods logic was added because the configuration is stored in a configMap and changes made to that configMap didn't trigger pods to be re-created when the Helm chart was redeployed (from Helm's perspective, nothing changed...so, no need to recreate the pods).  Deleting the pods forced them to re-read the configMap.  The "TO DO" note is suggesting a different approach of always undeploying the Helm release and re-deploying it.  That may be more attractive since the current behavior results in an extra round of pod deletions/recreations if there are changes that Helm does recognize.  The current approach works but looks "odd" if you're monitoring the pods and probably results in some wasted time/effort.

Reply:
Oh, then deleting the pods is better. Just remove the comment about removing the helm release. That sounds like a terrible way to go.